beets/dbcore/db.py

```diff
@@ -84,6 +85,10 @@ def _get_formatted(self, model, key):
 
         if self.for_path:
             sep_repl = beets.config['path_sep_replace'].as_str()
+            sep_drive = beets.config['drive_sep_replace'].as_str()
+            if re.match(r'^\w:', value):
+                value = value.replace(':', sep_drive)
```

Comment:
This is a little broad, for what it's worth: it will replace _all_ occurrences of `:`, where the only one we really need to replace is the first one. A positive lookbehind assertion might do the trick:

```py
>>> re.sub(r'(?<=^\w):', '_', 'f:o:o')
'f_o:o'
```

Reply:
Got it, I'll change that and add the extra test - however, the tests don't seem to be working on Python 2.7 :(

Reply:
Indeed; the error is this: `AttributeError: 'MoveTest' object has no attribute 'assertRegex'`

But I don't think that test needs `assertRegex`, and `assertEqual` or `assertIn` might suffice?

Reply:
Will the normal replacements be applied during move()? If so, that colon is getting replaced anyway.

Reply:
That seems to have done it.